src/tsolvers/lrasolver/FarkasInterpolator.h

```diff
@@ -42,16 +41,28 @@ struct DecomposedStatistics {
 
 class FarkasInterpolator {
 public:
-    FarkasInterpolator(LALogic & logic, vec<PtAsgn> const & explanations, std::vector<opensmt::Real> const & coeffs,
-                       std::map<PTRef, icolor_t> & labels) : FarkasInterpolator(logic, explanations, coeffs, &labels,
-                                                                                nullptr) {}
+    FarkasInterpolator(LALogic & logic, vec<PtAsgn> explanations, std::vector<opensmt::Real> coeffs,
+                       std::map<PTRef, icolor_t> labels)
```

Comment:
Is there a copy happening here?

Reply:
Yes, but that is intentional. We had this discussion about sink arguments taken by value vs by `&&`.
Or do you mean something else?

Reply:
Right, this is the place.